gsc/get_nauty.py

```diff
@@ -13,9 +13,9 @@ def qudit_graph_map(nx_wg, partition=None):
     For prime-power graph states, can colour by member or family.
     """
     # Gets list of all nodes by layer
-    us, vs, weights = zip(*nx_wg.edges.data('weight'))
+    us, vs, weights = list(zip(*nx_wg.edges.data('weight')))
```

Comment:
Why do we need to generate the whole list here? Same goes for all the other uses of `list` in this module

Reply:
Recasting the zip object as a list isn't necessary here but in some instances it is. I just included it here to be safe, but there's no need to include it here. I'll add comments to all the necessary ones.

Reply:
Great, thanks a bunch!